src/constants/constants.js

```diff
@@ -167,6 +167,7 @@ export const DIAL_PEERS_CONCURRENCY = 10;
 export const MIN_DIAL_FREQUENCY_MILLIS = 60 * 60 * 1000;
 
 export const PERMANENT_COMMANDS = [
+    'logPublicAddressesCommand',
```

Comment:
Please remove this from list of permanent commands, we don't need it right now, but we might need command in the future.

Reply:
Updated. Glad to know it worked